03.bowling/bowling.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames_tmp = shots.each_slice(2).to_a
+pins_of_tenth = frames_tmp.slice(9..-1)
+frame_of_tenth = []
+pins_of_tenth.map do |pin|
+  pin = [10] if pin == [10, 0]
+  frame_of_tenth += pin
+end
+frames = frames_tmp.slice(0..8)
+frames.push(frame_of_tenth)
```

Comment:
```suggestion
frames = [*frames_tmp.slice(0..8), frame_of_tenth]
```

こんな風にしても連結できるんじゃないかと思いますー。

Reply:
そのような記法があるんですね！！
(チェリー本、読み進めてるところなのでそのうち登場するかもしれませんが！)
勉強になります！！